src/layouts/AllServices.test.tsx

```
@@ -0,0 +1,223 @@
+}));
+
+jest.mock('../utils/common', () => ({
+  updateDocumentTitle: jest.fn(),
+}));
+
+jest.unmock('../components/NotificationsDrawer/DrawerPanelContent');
+
+jest.mock('@scalprum/react-core', () => ({
+  ScalprumComponent: (props: Record<string, unknown>) => (
+    <div data-testid="scalprum-content" data-scope={props.scope} />
+  ),
```

ESLint

[error] Replace `(⏎····<div·data-testid="scalprum-content"·data-scope={props.scope}·/>⏎··)` with `<div·data-testid="scalprum-content"·data-scope={props.scope}·/>`

(prettier/prettier)